stix2elevator/test/test_stepper.py

```diff
@@ -22,9 +23,24 @@
 _IGNORE = ()
 
 
+def custom_type(type_name):
+    return type_name not in [u"attack-pattern", u"campaign", u"course-of-action", u"grouping", u"identity", u"incident",
+                             u"indicator", u"infrastructure", u"intrusion-set", u"location", u"malware", u"malware-instance",
+                             u"note", u"observed-data", u"opinion", u"report", u"threat-actor", u"tool", "vulnerability",
+
+                             u"artifact", u"autonomous-system", u"directory", u"domain-name", u"email-addr", u"email-message",
+                             u"file", u"ipv4-addr", u"ipv6-addr", u"mac-address", u"mutex", u"network-traffic", u"process",
+                             u"software", u"url", u"user-account", u"windows-registry-key", u"x509-certificate",
+
+                             u"language_content", u"marking-definition", u"extension-definition", u"bundle"]
+
+
 def idiom_stepper_mappings(before_file_path, stored_json):
     """Test fresh conversion from XML to JSON matches stored JSON samples."""
     validator_options = stix2validator.parse_args("")
+    initialize_options()
+    set_option_value("missing_policy", "use-extensions")
+    set_option_value("custom_property_prefix", "elevator")
```

Comment:
I think we can follow a similar design pattern to the test_idioms.py tests were we use Environment Variables to pass options for the tests instead of hard-coding them here.

Reply:
For testing "elevator" is a good default.  Are you thinking that we should test for weird custom_property_prefixies?

As I stated in the last comment, I do not think the stepper should produce anything but CS02 without anything deprecated.

Reply:
Yes, this default is good for test_stepper.py, I should have made this comment on the resolved suggestion I made above for the stepper CLI argument. Can you point me to the comment about

> I do not think the stepper should produce anything but CS02 without anything deprecated.